container-engine-lib/lib/backend_impls/docker/object_attributes_provider/label_key_consts/label_key_consts.go

```diff
@@ -66,3 +67,4 @@ var PrivateIPDockerLabelKey = docker_label_key.MustCreateNewDockerLabelKey(priva
 var UserServiceGUIDDockerLabelKey = docker_label_key.MustCreateNewDockerLabelKey(userServiceGuidDockerLabelKeyStr)
 var LogsEnclaveUUIDDockerLabelKey = docker_label_key.MustCreateNewDockerLabelKey(logsEnclaveUuidLabelKeyStr)
 var LogsServiceUUIDDockerLabelKey = docker_label_key.MustCreateNewDockerLabelKey(logsServiceUuidDockerLabelKey)
+var EnclaveModeLabelKey = docker_label_key.MustCreateNewDockerLabelKey(enclaveTypeLabelKeyStr)
```

Comment:
nit: is a bit confusing for me that the var name says `EnclaveMode` but the key says `enclave-type`, I think that these could be aligned WDYT?

Reply:
Good catch, this is unused